docs/persistentvolumeclaim.md

```diff
@@ -0,0 +1,149 @@
+## PersistentVolumeClaim Conformance matrix
+
+Following matrix shows supported PersistentVolumeClaim parameters for lvm-localpv.
+
+| PersistentVolumeClaim Parameters | LVM CSI Driver | E2E Coverage |
+| -------------------------------- | -------------- | ------------ |
+| [AccessMode](#accessmode) <br> Supported access modes are <li> ReadWriteOnce </li> </br> | Supported | [Yes](https://github.com/openebs/lvm-localpv/tree/master/e2e-tests/experiments/lvm-localpv-provisioner#readme) |
+| [Storageclass](#storageclassname) | Supported | [Yes](https://github.com/openebs/lvm-localpv/tree/master/e2e-tests/experiments/lvm-localpv-provisioner#readme) |
+| [Capacity Resource](#capacity-resource) | Supported | Yes |
+| [VolumeMode](#volumemode) <br> Supported volume modes are <li> Block </li> <li> Filesystem </li> </br> | Supported | Yes <br> Test exist only for Filesystem mode </br>|
```

Comment:
let us say "Test cases available Filesystem"?

Reply:
Updated the doc